api/app/src/command/medical/document/document-download.ts

```diff
@@ -1,9 +1,96 @@
+import { makeLambdaClient } from "../../../external/aws/lambda";
 import { makeS3Client } from "../../../external/aws/s3";
 import { Config } from "../../../shared/config";
+import { DocConversionType } from "../../../routes/medical/schemas/documents";
 
+const lambdaClient = makeLambdaClient();
 const s3client = makeS3Client();
 
-export const downloadDocument = async ({ fileName }: { fileName: string }): Promise<string> => {
+const htmlConversionType = "html";
+const pdfConversionType = "pdf";
+
+export const downloadDocument = async ({
+  fileName,
+  conversionType,
+}: {
+  fileName: string;
+  conversionType?: DocConversionType;
+}): Promise<string> => {
+  const { exists, contentType } = await doesObjExist({ fileName });
+
+  if (!exists) throw new Error("File does not exist");
+
+  if (contentType !== "application/xml" && contentType !== "text/xml")
```

Comment:
shouldn't this be:

 if (conversionType && contentType !== "application/xml" && contentType !== "text/xml")